### PR TITLE
doc: add minutes for meeting 22 Apr 2020

### PR DESCRIPTION
**Pull request by gireeshpunathil**
Fixes: https://github.com/nodejs/diagnostics/issues/381

/cc @nodejs/diagnostics

### CODE REVIEW

File: wg-meetings/2020-04-22.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+
+* COVID-19 and WG sittings [#370](https://github.com/nodejs/diagnostics/issues/370)
+  * No concerns/comments this week
+
+* proposal: elevate diagnostic report to tier1 [#369](https://github.com/nodejs/diagnostics/issues/369)
+  * Gireesh: Process was approved, raised in main repo now
+  * After waiting for seven days we could process, but we have only one approval at the moment
+  * Michael: did I approve?
+  * Gireesh: une comment about putin back old report to unclassified
```
Comment (legendecas) [inline wg-meetings/2020-04-22.md:51]:
```suggestion
  * Gireesh: one comment about putting back the old report to unclassified
```

Comment (gireeshpunathil) [reply wg-meetings/2020-04-22.md]: @legendecas - thanks, fixed!

File: wg-meetings/2020-04-22.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+
+* COVID-19 and WG sittings [#370](https://github.com/nodejs/diagnostics/issues/370)
+  * No concerns/comments this week
+
+* proposal: elevate diagnostic report to tier1 [#369](https://github.com/nodejs/diagnostics/issues/369)
+  * Gireesh: Process was approved, raised in main repo now
+  * After waiting for seven days we could process, but we have only one approval at the moment
+  * Michael: did I approve?
+  * Gireesh: une comment about putting back old report to unclassified
```
Comment (HarshithaKP) [inline wg-meetings/2020-04-22.md:51]:
```suggestion
  * Gireesh: One comment about putting back the old report to unclassified
```

Comment (gireeshpunathil) [reply wg-meetings/2020-04-22.md]: @HarshithaKP - done, fixed the typos.

**Comment by HarshithaKP** [comment]
Landed in bba0ac057a312309d9567584ee22f7b09037c461

**Pull request closed without merging**